### PR TITLE
Individual reminder

### PR DESCRIPTION
**Pull request by mjvalade**
@stevekinney @brittanystoroz @tehviking @martensonbj 

## Purpose
Nested the `Reminder` route in the `Reminders` route in order to allow the user to see the selected reminder in the Reminders List.

## Approach
Reminder titles and dates are rendered to a list on the homepage. When user clicks on title the full reminder, including notes, renders to the same page. The URL is updated to reflect which reminder is selected. Styling has been added to reflect which reminder is active.

### Learning

We referenced these resources to understand routing and links.
[Beard Beats Two class exercise](https://github.com/turingschool-examples/beard-beats-two)
[Ember Routing Guide](https://guides.emberjs.com/v2.8.0/routing/defining-your-routes/)

### Test coverage 

Both acceptance tests are passing.

### Follow-up tasks

Receive Issue #3

### Screenshots
Action Video

Closes #2

### CODE REVIEW

File: app/routes/reminders.js [hunk 1]
```diff
@@ -2,6 +2,6 @@ import Ember from 'ember';
 
 export default Ember.Route.extend({
   model() {
-    return this.get('store').findAll('reminder');
+    return this.store.findAll('reminder');
```
Comment (stevekinney) [inline app/routes/reminders.js]: Good catch!

File: app/templates/reminders.hbs [hunk 2]
```diff
@@ -1,16 +1,7 @@
...
-      {{#if reminder.notes}}
-        <p>{{reminder.notes}}</p>
-      {{/if}}
-
-      <button type="'button'" name="button">Pin Reminder</button>
-    </div>
+    <h1>{{#link-to 'reminders.reminder' reminder.id class="spec-reminder-item"}}{{reminder.title}}{{/link-to}}</h1>
+    <p>{{reminder.date}}</p>
   {{/each}}
+  {{outlet}}
```
Comment (stevekinney) [inline app/templates/reminders.hbs]: This was a good move.

File: app/templates/reminders/reminder.hbs [hunk 3]
```diff
@@ -0,0 +1,10 @@
+<div class="reminder-details">
+  <h1 class="spec-reminder-title">{{model.title}}</h1>
+  <p>{{model.date}}</p>
+
+  {{#if model.notes}}
```
Comment (stevekinney) [inline app/templates/reminders/reminder.hbs]: I like that you're not showing the notes element if there isn't one. 👍

File: app/routes/reminders.js (hunk 1 above)
Comment (martensonbj) [inline app/routes/reminders.js]: After digging into this, we've agreed that the original syntax (`this.get('store')...`) is the correct way to get at this information.[ Most recent docs](https://guides.emberjs.com/v2.8.0/routing/specifying-a-routes-model/) support this as well.

**Pull request merged**